Components/button/button.html

```diff
@@ -0,0 +1,237 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link href="https://fonts.googleapis.com/icon?family=Material+Icons" rel="stylesheet">
+    <link rel="stylesheet" href="../library.css">
+    <link rel="stylesheet" href="../../reset.css">
+    <link rel="stylesheet" href="../button/button.css">
+    <title>DarkBucket</title>
+    <link rel="icon" href="images/ICON.png" type="image/x-icon">
+
+
+</head>
+
+<body>
+    <header>
+        <nav class="nav">
+            <div class="nav__left">
+                <a href="../../Index.html">
+                    <div class="nav__logo">
+                        <img src="../../images/DARKBUCKET-title.svg" class="nav__logo">
+                    </div>
+                </a>
+            </div>
+            <div class="nav__right">
+                <div class="social-media-list">
+                    <ul class="social-media-list">
+                        <li class="social-media-items"><a href="#"><img class="icon-height"
+                                    src="../../images/Linkedin.png" alt="Linkedin"></a></li>
```

Comment:
As from your folder structure I think you can directly go like this --> /images/linkdin.png
same you can check in others.

Reply:
tried resolving this with your method but it wasn't working for me as I have my file structure like this!

![image](https://user-images.githubusercontent.com/91378157/153698528-0acfe884-88d4-4024-83a3-b0e4c8e336a9.png)
all components are in components and images are in images according to component folder
![image](https://user-images.githubusercontent.com/91378157/153698629-ed99e949-4b83-4993-977a-606fc650f3c1.png)




In Components it looks like this  each component is in new folder
![image](https://user-images.githubusercontent.com/91378157/153698690-0be99a19-96b3-41ba-80fd-c66eedacfc24.png)


and in button

![image](https://user-images.githubusercontent.com/91378157/153698707-9451bf55-f0d6-4a44-9ffc-2f1ea9c22692.png)



so to use any image i have to come two folders back to access images!
![image](https://user-images.githubusercontent.com/91378157/153698737-b4b0e34d-d29d-4be5-8e6a-93d795ca4eb0.png)


thats why using src="../../images/Linkedin.png"

Reply:
cool then go ahead with current structure if that is the case!